Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -102,10 +102,10 @@ class EventNotificationsManager {
                 // don't send the notification if there isn't
                 let placesDB = PlacesProvider()
                 placesDB.place(forKey: event.placeId) { place in
-                    guard let _ = place else { return  NSLog("Not sending notification for \(event.description) as it has no place") }
+                    guard let _ = place else { return  NSLog("Not sending notification for \(event.id) as it has no place") }
```

Comment:
fwiw, liuche is changing these to md5 hex hashes so it'll be hard to debug without querying Firebase directly – perhaps we should get a team python REPL going for Firebase, or something (but someone must have done that already!)?